.circleci/config.yml

```diff
@@ -61,16 +61,6 @@ commands:
           paths:
             - node_modules
 
-  codecov:
-    parameters:
-      flag:
-        type: string
-    steps:
-      - run:
-          name: Upload code coverage reports to Codecov
-          # do not upload if screenshotter tests are skipped
-          command: ./node_modules/.bin/codecov -F "<< parameters.flag >>"
```

Comment:
I'm not sure there is need to separate Jest tests coverage and screenshotter coverage.